charts/seed-monitoring/charts/prometheus/rules/kube-etcd3.rules.yml

```diff
@@ -2,19 +2,30 @@ groups:
 - name: kube-etcd3.rules
   rules:
 
-  # alert if another failed peer will result in an unavailable cluster
-  - alert: KubeEtcd3InsufficientPeers
-    expr: absent(up{job="kube-etcd3",role="main"})
-      or absent(up{job="kube-etcd3",role="events"})
-      or count(up{job="kube-etcd3"} == 0) by(role) > (count(up{job="kube-etcd3"}) by (role) / 2 - 1)
+  # alert if etcd is down
+  - alert: KubeEtcd3Down
+    expr: absent(up{job="kube-etcd3",role="main"}) or absent(up{job="kube-etcd3",role="events"})
```

Comment:
Just for me (not being too familiar with the alert definition language): What is this `job="kube-etcd"` standing for?

Reply:
In Prometheus terms, an endpoint you can scrape is called an instance, usually corresponding to a single process. A collection of instances with the same purpose, a process replicated for scalability or reliability for example, is called a job.

For example, an API server job with four replicated instances:

    job: api-server
        instance 1: 1.2.3.4:5670
        instance 2: 1.2.3.4:5671
        instance 3: 5.6.7.8:5670
        instance 4: 5.6.7.8:5671

Automatically generated labels and time series

When Prometheus scrapes a target, it attaches some labels automatically to the scraped time series which serve to identify the scraped target:

    job: The configured job name that the target belongs to.
    instance: The <host>:<port> part of the target's URL that was scraped.

`up{job="kube-etcd3"}` means - find a time series metric `up` for which the value of the `job` label is `kube-etcd3`

Reply:
OK, thank you for that explanation. And [here](https://github.com/gardener/gardener/blob/d318acd0ba5099a94b2c1d324af55533c060d228/charts/seed-controlplane/charts/etcd/templates/etcd-statefulset.yaml#L33) the job name is defined on the etcd service. That's how Prometheus knows which pods/services are relevant. Thanks!